examples/test.html

```diff
@@ -106,9 +106,6 @@
 
 
   $.init(); // init(['collisions', 'sound', 'astar', ...])
-  // Bind keyboard
-  $.input.bind([ENTER, KEY_A, LEFT, RIGHT, UP, DOWN]);
-  $.cam.setWorldSize(1024, 1024);
```

Comment:
I mentioned it in some comments above, binding and camera setup should be part if the game itself, since those parameters are not fixed. Every game has different world size and different key bindings.

Reply:
I'm proposing `build` methods only as prosthetics for actual builder pattern. Merely a vehicle to make relationships explicit. Other than that I agree - I'd abstract out only necessary parameters and let the game initialize them however it wants.

Reply:
👍